#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_beatextractor.sh-path.dpatch by  <leleobhz@localhost>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: An patch to correct places for the debian install

@DPATCH@

--- ./BeatExtractor.sh.orig	2007-02-23 16:25:00.000000000 -0200
+++ ./BeatExtractor.sh	2007-02-23 16:26:06.000000000 -0200
@@ -56,7 +56,7 @@
 fi
 
 ## Setting commander to 0%
-`pwd`/Progress.sh "$PrBarObj" "0"
+/usr/share/BeatExtractor/Progress.sh "$PrBarObj" "0"
 
 # TODO01
 # rm -rf *.beat.wav *filt.wav *.orig-beat-mix.wav
@@ -122,7 +122,7 @@
 file$ = $file
 fil$ = path$ + "/" + file$
 
-progressbar$ = environment$ ("PWD") + "/Progress.sh"
+progressbar$ = "/usr/share/BeatExtractor/Progress.sh"
 
 ##
 # mindur is the minimum duration allowed between two consecutive boundaries
